docs/tutorial/workers.md

```diff
@@ -50,10 +54,7 @@ graph TD
     worker --> |creates| flow_run_infra
 ```
 
-!!! note "Security Note"
```

Comment:
Moved this to the deployment page of the tutorials since its generally true of our deployment interface and not just a feature of workers and work pools.